### PR TITLE
[DO-1934] check property right

### PR DESCRIPTION
**Pull request by EmmaSamoilova**

### CODE REVIEW

File: business_register/converter/declaration.py [hunk 1]
```diff
@@ -49,7 +49,8 @@ def __init__(self):
             '[Не відомо]',
             "[Член сім'ї не надав інформацію]",
             '[Конфіденційна інформація]',
-            'Не визначено'
+            'Не визначено',
+            'невідомо'
```
Comment (roman-tiukh) [inline business_register/converter/declaration.py:53]: trailing comma

File: business_register/converter/declaration.py [hunk 2]
```diff
@@ -905,24 +906,32 @@ def save_property_right(self, property, acquisition_date, rights_data):
             country_of_citizenship_info = data.get('citizen')
             # TODO: return country
             if country_of_citizenship_info:
+                if country_of_citizenship_info == 'Громадянин України':
+                    country_of_citizenship_info = '1'
```
Comment (roman-tiukh) [inline business_register/converter/declaration.py:910]: what is `1`? maybe create constant?

File: business_register/converter/declaration.py (hunk 1 above)
Comment (EmmaSamoilova) [reply business_register/converter/declaration.py]: done

File: business_register/converter/declaration.py (hunk 2 above)
Comment (EmmaSamoilova) [reply business_register/converter/declaration.py]: done

**Review by roman-tiukh — APPROVED** [review]

File: business_register/converter/declaration.py [hunk 3]
```diff
@@ -885,44 +889,53 @@ def save_property_right(self, property, acquisition_date, rights_data):
...
                     pass
                 else:
                     pep = self.find_person(owner_id)
             other_owner_info = data.get('rights_id')
             if not pep and other_owner_info:
-                if other_owner_info == '1':
+                if other_owner_info == self.DECLARANT:
                     pep = property.declaration.pep
-                elif other_owner_info == 'j':
+                elif other_owner_info == self.OTHER_PERSON:
```
Comment (IegorSoboliev) [inline business_register/converter/declaration.py:903]: I think the time to decide has come)). Roma, should we we store data like 'Власником є третя особа'?

File: business_register/converter/declaration.py [hunk 4]
```diff
@@ -874,6 +877,7 @@ def save_property_right(self, property, acquisition_date, rights_data):
              'ЗУ «Про запобігання корупції»'): PropertyRight.BENEFICIAL_OWNERSHIP,
             "[Член сім'ї не надав інформацію]": PropertyRight.NO_INFO_FROM_FAMILY_MEMBER,
         }
+        UKRAINE_NACP_ID = '1'
```
Comment (IegorSoboliev) [inline business_register/converter/declaration.py:880]: hardcoding))

File: business_register/converter/declaration.py [hunk 5]
```diff
@@ -885,44 +889,53 @@ def save_property_right(self, property, acquisition_date, rights_data):
...
+            middle_name = data.get('ua_middlename') if data.get('ua_middlename') not in self.NO_DATA else ''
+            ukr_full_name = data.get('ukr_fullname')
+            eng_full_name = data.get('eng_fullname')
             if (
                     last_name not in self.NO_DATA
                     or first_name not in self.NO_DATA
                     or middle_name not in self.NO_DATA
             ):
                 full_name = f'{last_name} {first_name} {middle_name}'
+            elif ukr_full_name and ukr_full_name not in self.NO_DATA:
```
Comment (IegorSoboliev) [inline business_register/converter/declaration.py:929]: None value already is in self.NO_DATA)

**Review by IegorSoboliev — APPROVED** [review]

File: business_register/converter/declaration.py (hunk 5 above)
Comment (EmmaSamoilova) [reply business_register/converter/declaration.py]: done

**Pull request merged**